Build/PreBuild.ps1

```diff
@@ -86,8 +85,8 @@ try {
         $packageJson | ConvertTo-Json -depth 8 | Set-Content $packageJsonFile
     }
 
-    # install pylance
-    npm install
+    # force install latest pylance, always grab latest even if a lower version is already installed
+    npm install --force
```

Comment:
So what this force install does is to omit the .\build\nuget locals all -clear step? Is my understanding correct?

Reply:
No, clearing the nuget cache is for nuget only. This is specifically for npm only :)

Reply:
This way fails the build because the warning from npm writes to the error stream. So instead, my latest change checks for the `node_modules` folder and deletes it if it exists